docs/design.html

```diff
@@ -298,9 +298,12 @@
     In distributed systems terminology we only attempt to handle a "fail/recover" model of failures where nodes suddenly cease working and then later recover (perhaps without knowing that they have died). Kafka does not
     handle so-called "Byzantine" failures in which nodes produce arbitrary or malicious responses (perhaps due to bugs or foul play).
     <p>
-    A message is considered "committed" when all in sync replicas for that partition have applied it to their log. Only committed messages are ever given out to the consumer. This means that the consumer need not worry
-    about potentially seeing a message that could be lost if the leader fails. Producers, on the other hand, have the option of either waiting for the message to be committed or not, depending on their preference for
-    tradeoff between latency and durability. This preference is controlled by the acks setting that the producer uses.
+    We can now more precisely define that a message is considered committed when all in sync replicas for that partition have applied it to their log.
+    Only committed messages are ever given out to the consumer. This means that the consumer need not worry about potentially seeing a message that could be lost if the leader fails. Producers, on the other hand, 
+    have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This preference is controlled by the acks setting that the 
+    producer uses.
+    Note that the producer's acks setting also determines whether a message can be committed or not. Topics have a setting for a "minimum number" of in sync replicas, that is checked only when the producer requests 
```

Comment:
> Note that the producer's acks setting also determines whether a message can be committed or not.

This sentence could be clarified a little. I think the misleading aspect of min.isr that you are trying to capture is the fact that it only guards writes from producers with acks=all. However, once a message is written to the log, whether or not it can become "committed" is independent of producer acks or min.isr.